app/views/internal/organizations/_activity.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="row">
+  <h3><u>Activity</u></h3>
+  <ul>
+    <li><%= @organization.articles.size %> articles</li>
+    <li><%= @organization.followers.size %> followers</li>
```

Comment:
This is kind of interesting, if we call `size` on articles it will return a count query from MySQL, however, on followers it loads all of the records 😬While this is not a huge deal to start, popular orgs might be very slow to load this way.
```
[40] pry(main)> o.articles.size
   (0.6ms)  SELECT COUNT(*) FROM "articles" WHERE "articles"."organization_id" = $1  [["organization_id", 3]]
=> 0
[41] pry(main)> o.followers.size
  Follow Load (0.4ms)  SELECT "follows".* FROM "follows" WHERE "follows"."followable_id" = $1 AND "follows"."followable_type" = $2 AND "follows"."blocked" = $3  [["followable_id", 3], ["followable_type", "Organization"], ["blocked", false]]
=> 0
```